ExamGenerator.Web.InputModels/Question/QuestionInputModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ExamGenerator.Services.Mapping;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace ExamGenerator.Web.InputModels.Question
+{
+    public class QuestionInputModel : IMapTo<Data.Models.Question>
+    {
+        [Required]
+        public string Content { get; set; }
+
+        [Required]
```

Comment:
It is not necessary to have [Requiered] becuase int is value type and cannot be null